src/common/darktable.h

```diff
@@ -173,9 +173,16 @@ typedef enum dt_debug_thread_t
   DT_DEBUG_CAMERA_SUPPORT = 1 << 16,
 } dt_debug_thread_t;
 
+typedef struct dt_codepath_t
+{
+  unsigned int SSE2 : 1;
+  unsigned int _no_intrinsics : 1;
```

Comment:
i just wanted to visually separate this element from others in this struct.
(all others are supposed to be intrinsics sets, this - if any of those is available)